### PR TITLE
Update e2e test for new model training configs

### PR DESCRIPTION
**Pull request by oliverwm1**
PR on top of #1304

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -147,8 +147,7 @@ jobs:
           name: Install dependencies
           command: |
             apt-get install -y wget jq make gettext gzip
-            wget -qO /bin/yq https://github.com/mikefarah/yq/releases/download/3.2.1/yq_linux_386
-            chmod +x /bin/yq
+            pip3 install yq
```
Comment (mcgibbon) [inline .circleci/config.yml:150]: Do we need/want a version pin on this?

File: tests/end_to_end_integration/.gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 config.yaml
 kustomization.yaml
+training-data-config-compiled.yaml
+validation-data-config-compiled.yaml
```
Comment (mcgibbon) [inline tests/end_to_end_integration/.gitignore:4]: nit: We normally name our files with underscores and argo variables with dashes. You may want to refactor these to use underscores.

**Review by mcgibbon — APPROVED** [review]
LGTM. Minor nits to consider.

File: tests/end_to_end_integration/.gitignore (hunk 2 above)
Comment (oliverwm1) [reply tests/end_to_end_integration/.gitignore:4]: for better or worse, that has not been the standard practice for config/argo yamls. E.g. see the workflows/argo directory or the various config yamls in vcm-workflow-control. I'm going to leave this as is.

File: .circleci/config.yml (hunk 1 above)
Comment (oliverwm1) [reply .circleci/config.yml]: Okay I pinned it at a recent version.

**Pull request merged**